alsaaudio.c

```diff
@@ -1051,6 +1051,7 @@ alsapcm_read(alsapcm_t *self, PyObject *args)
 {
 	int res;
 	int size = self->framesize * self->periodsize;
+	int sizeout;
```

Comment:
This could be initialized to 0

Reply:
Initialize to zero and then only set it when res > 0?

Reply:
Yes, that's what I would have done too. Thanks!